walk-in-interview/src/main/webapp/update-job/script.js

```diff
@@ -14,43 +14,47 @@ const CurrentLocale = 'en';
 import {AppStrings} from '../strings.en.js';
 import {API} from '../apis.js';
 
-import {JOB_ID_PARAM, getCookie, getRequirementsList, 
+import {JOB_ID_PARAM, getRequirementsList,
   setErrorMessage, renderSelectOptions} from '../common-functions.js';
 
 const STRINGS = AppStrings['job'];
 const UPDATE_JOB_STRINGS = AppStrings['update-job'];
 const COMMON_STRINGS = AppStrings['common'];
-const HOMEPAGE_PATH = '../job-details/index.html';
+const HOMEPAGE_PATH = '../index.html';
```

Comment:
are u sure when u update, it will return back to the home page with all the job listings, not the job detail page of that job?